src/i18n_check/utils.py

```diff
@@ -1,6 +1,256 @@
 """
-Things to include:
-    - files_to_check
+Utility functions for the i18 check keys.
 """
 
+import json
+import os 
+import re
+from pathlib import Path
+import yaml
+from collections.abc import MutableMapping
+import glob
+import string
+
+
+
+# MARK: YAML Reading
+
+# Define the path to the YAML configuration file
+config_path = Path(__file__).parent.parent.parent / ".i18n-check.yaml"
+
+with open(config_path, "r", encoding="utf-8") as file:
+    config = yaml.safe_load(file)
+
+# Define Globals
+
+
+i18n_check_dir = str(Path(config["i18n-dir"]).resolve())
+json_file_directory = Path(config["i18n-dir"]).resolve()
+frontend_directory = Path(config["src-dir"]).resolve()
+en_us_json_file = Path(config["i18n-src"]).resolve()
```

Comment:
Let's remember that en-US might not be the src language for other projects, and also we might expand this out to work for other file types eventually :) Will rename this accordingly.